scripts/python/api/tnode.py

```diff
@@ -87,22 +94,18 @@ def enableV2Plugin(self, role="master", args=""):
     # Start netplugin process on vagrant node
     def startNetplugin(self, args=""):
         ssh_object = self.sshConnect(self.username, self.password)
-        if os.environ["CONTIV_CLUSTER_STORE"].startswith("etcd"):
-            store_args = " --etcd-endpoints %s " % os.environ["CONTIV_CLUSTER_STORE"].replace("etcd", "http")
-        else:
-            store_args = " --consul-endpoints %s " % os.environ["CONTIV_CLUSTER_STORE"].replace("consul", "http")
-
-        # TODO: determine the fwdmode and netmode
-        fwd_arg = " --fwdmode bridge "
-        net_arg = " --netmode vlan "
-        command = "sudo " + self.binpath + "/netplugin --vlan-if eth2 --vlan-if eth3 " + fwd_arg + fwd_arg + store_args + args + "> /tmp/netplugin.log 2>&1"
+        # NOTE: this testing only used in mesos-docker
+        mode_args = " --fwdmode bridge --netmode vlan --mode docker "
+        command = "sudo " + self.binpath + "/netplugin --vlan-if eth2 --vlan-if eth3 " + mode_args + gen_cluster_store_args() + args + "> /tmp/netplugin.log 2>&1"
```

Comment:
wrap?

Reply:
i'd make it in different PR, codes under systest are pretty sick